src/client/voice/VoiceBroadcast.js

```diff
@@ -67,8 +67,8 @@ class VoiceBroadcast extends EventEmitter {
    */
   end() {
     for (const dispatcher of this.dispatchers) this.delete(dispatcher);
-    const index = this.client.voice.broadcasts.indexOf(this);
-    if (index !== -1) this.client.voice.broadcasts.splice(index, 1);
+    const index = this.client.broadcasts.indexOf(this);
```

Comment:
Can you rename the client variable to voiceManager, including updating the jsdocs in the constructor? That way it's correctly showing what it is

Reply:
Or pass client when constructing `VoiceBroadcast`:
https://github.com/discordjs/discord.js/blob/f479afca8212e00208799793aac0286553669106/src/client/voice/ClientVoiceManager.js#L40

Reply:
i think its better to pass the client

Reply:
just change the pull to change the this.client 
`const broadcast = new VoiceBroadcast(this.client); `